code-experiments/src/coco_problem.c

```diff
@@ -29,6 +29,9 @@
 void coco_evaluate_function(coco_problem_t *problem, const double *x, double *y) {
   /* implements a safer version of problem->evaluate(problem, x, y) */
   size_t i, j;
+  size_t is_feasible;
```

Comment:
A small comment: size_t doesn't look the right type for the is_feasible flag. coco_is_feasible returns an int, which seems perfectly fine to me. Assignment to a size_t should actually give a compiler warning that a sign may be lost.